MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -606,6 +624,16 @@ - (void)refreshEventListeners:(NSArray *)liveEventTypesFilterForMessages
                 [self didReceiveReceiptEvent:event roomState:roomState];
             }
         }];
+        
+        // Flair handling: register a listener for the related groups state event in this room.
+        relatedGroupsListener = [_timeline listenToEventsOfTypes:@[kMXEventTypeStringRoomRelatedGroups] onEvent:^(MXEvent *event, MXTimelineDirection direction, MXRoomState *roomState) {
+            
+            if (MXTimelineDirectionForwards == direction)
+            {
+                // The flair settings have been updated: flush the current bubble data and rebuild them.
+                [self reload];
```

Comment:
is it the answer to my question in the other comment?

Reply:
Not really, here we detect that the related groups have been updated. We reload the data source because all the room bubbles have to update their sender flair.